code/obj/item/misc_wizard_stuff.dm

```diff
@@ -220,6 +220,85 @@
 			src.do_brainmelt(user, 2)
 			return
 
+/obj/item/staff/thunder
+	name = "stave of thunder"
+	desc = "A staff sparkling with static electricty. Who's afraid of a little thunder?"
+	icon_state = "staffthunder3"
+	item_state = "staffthunder"
+	var/thunder_charges = 3
+
+	New()
+		. = ..()
+		START_TRACKING
+
+	disposing()
+		. = ..()
+		STOP_TRACKING
+
+	pixelaction(atom/target, params, mob/user, reach)
+		if(!IN_RANGE(user, target, 10))
+			return
+		if (istype(get_area(target), /area/station/chapel) || istype(get_area(user), /area/station/chapel))
+			boutput(user, "<span class='alert'>You cannot summon lightning on holy ground!</span>") //phrasing works if either target or mob are in chapel heh
+			return
+		if (thunder_charges <= 0)
+			boutput(user, "<span class='alert'>[name] is out of charges! Magically recall it to restore it's power.</span>")
+			return
+		if (iswizard(user))
+			thunder_charges -= 1
+			var/turf/T = get_turf(target)
+			var/obj/lightning_target/lightning = new/obj/lightning_target(T)
+			playsound(T, "sound/effects/electric_shock_short.ogg", 70, 1)
+			lightning.caster = user
+			UpdateIcon()
+			flick("[icon_state]_fire", src)
+			..()
+
+	attack_hand(var/mob/user as mob)
+		if (user.mind)
+			if (iswizard(user) || check_target_immunity(user))
+				if (user.mind.key != src.wizard_key && !check_target_immunity(user))
+					boutput(user, "<span class='alert'>The [src.name] is magically attuned to another wizard! You can use it, but may not summon it magically.</span>")
+				..()
+				return
+			else
+				boutput(user, "<span class='alert'>Your hand phases through the [name]'s handle, like it's not there...</span>")
```

Comment:
idk - i think shocking for 1 damage or something would be nice, otherwise people might just keep clicking it and not seeing the message because there's no non-text feedback

Reply:
ditto for below stuff

Reply:
Agreed. Added a new proc that's called in the place of all of these, which deals 1 second of disorientate and plays the classic sparks4.ogg . I wanted to keep it away from doing real damage to people trying to pick it up, I think this is good enough feedback to help clear up that it's not just lag or something. Also doesn't disorientate if you have SMES human of course